### DISCUSSION TITLE
how to purchase hosting in vercel

### DISCUSSION THREAD

**Discussion by naveenkhatanagit** [vercel/community#3873] [Help] [answered]
### Summary

hello,i have recently developed site with nextjs.can you help me about pricing of vercel

### Example

https://awesmaticfrontend.vercel.app/

### Steps to Reproduce

hello,i have recently developed site with nextjs.can you help me about pricing of vercel

**Comment by cpeaustriajc** [comment, marked as answer]
https://vercel.com/pricing
